Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -247,6 +265,24 @@ class PaymentSheetTestPlayground: UIViewController {
         return configuration
     }
 
+    var intentConfig: PaymentSheet.IntentConfiguration {
+
+        switch intentMode {
+        case .payment:
+            return PaymentSheet.IntentConfiguration(mode: .payment(amount: 1000, currency: "USD",
+                                                                   setupFutureUsage: nil),
+                                                                captureMethod: .automatic)
```

Comment:
All of these properties and payment_method_types should match what the toggles say. 
e.g. if the toggle says 'currency = EUR', the currency here should be 'EUR'.

In practice it's probably simpler to not look at the toggles to determine the values, but instead look at the Intent returned by our backend.

Reply:
Good catch, I changed this to look at the currency toggle.

I figure we shouldn't be looking at an intent returned by our sever for the deferred workflow 🙈

Reply:
Currency was just one example

- amount, setup_future_usage, capture method needs to match the values on the Intent 
- p_m_types needs to match the Intent if automatic pm types is off

Reply:
Still doesn't make sense to me... my understanding is in a real-world deferred case the user would not create an intent until confirm time. Why would we make the playground create an intent at loading time (and read it), then presumably create another intent at confirm time? I think the most correct thing here to do would not even make a request to glitch to create an intent when we load and only make a request to glitch when we confirm. The only reason I left the loading in there was because the entire state of the playground is so tied to that function (`load`) e.g. re-enabling toggles. Maybe we need to change that to not even create an intent when we load in the deferred case to closely match the real world use case.

re: p_m_types, shouldn't that come from the elements session response or the `IntentConfiguration.paymentMethodTypes`, why would we read that off the intent?